src/Guards/TokenGuard.php

```diff
@@ -131,7 +131,7 @@ public function client(Request $request)
             );
         } elseif ($request->cookie(Passport::cookie())) {
             if ($token = $this->getTokenViaCookie($request)) {
-                return $this->clients->findActive($token['aud']);
+                return $this->clients->findActive($token['aud'][0]);
```

Comment:
Why was this change needed?

Reply:
The Lcobucci library breaks up audiences so you can have multiple values instead of a single string. When it parses the token it always produces an array. This is the default behaviour anyways. I don't know if there is potentially a way to stop that. It could in the Encoder class the audience should just be imploded into a string to replicate the way Firebase presents that value. I can do a bit more research into that.

I'll admit it as it stands it for people extending the method of the TokenGuard, then this change could be problematic for just a minor release.